shared/js/participantStore.js

```diff
@@ -119,6 +120,19 @@ loop.store.ParticipantStore = function() {
         }
       }
 
+      if (!updatedParticipant.hasNotifiedOfJoin &&
+          updatedParticipant.isHere &&
+          updatedParticipant.participantName &&
```

Comment:
Are we always guaranteed a name? Maybe have a fallback just in case someone does something nasty...